src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const queryChange = (value: string) => {
+    setQuery(value);
+  };
+
+  let visibleMovies = moviesFromServer;
+
+  if (query.length) {
```

Comment:
let's turn it into ternary operator, try not to reassign variables
```suggestion
  const visibleMovies = query.length > 0
    ? filter here
    : moviesFromServer
```